packages/bot/src/scratch/index.js

```diff
@@ -81,6 +81,15 @@ export const runBot = (limitations = {}) => {
             block.setDisabled(true);
         }
     });
+
+    const { mandatoryMainBlocks } = config;
+    const has_main_blocks = mandatoryMainBlocks
+        .every(block => top_blocks.filter(top_block => top_block.type === block).length);
+
+    if (!has_main_blocks) {
+        globalObserver.emit('Error', new Error('Root Block(s) missing from workspace'));
```

Comment:
- Please check with copywriting for this string
- Use `translate` to translate strings

Reply:
As we discussed last time, we shouldn't translate error message right?

Reply:
Errors that are sent to TrackJS shouldn't be translated, errors shown to users should be translated.